### ISSUE TITLE
Sidenav issues on Android (menu-toggles do not open)

### ISSUE THREAD

**Issue by WhiteAbeLincoln** [angular/material#3146]
When viewing the sidenav on android, the toggles never open after tapping on them. The animation still works, but the menu-links below the toggles never show up.

This issue shows up on chrome and firefox for android, on multiple devices. Older versions of the docs work, but on documentation version 0.9.8-RC1, the links below the toggle don't show.

This is probably due to changes in the css between 0.9.7 and 0.9.8-rc1, since I couldn't find any other javascript changes between 0.9.8 and 0.9.7. It could be something to do with the way android handles the css
The `.menu-toggle-list` transition property changed from `max-height` to `height` (line 316 in docs/app/css/style.css), and 
```css
.menu-toggle-list.ng-hide {
  max-height: 0;
}
```
changed to 
```css
.menu-toggle-list.ng-hide {
  height: 0;
}
```
This seems most likely to be the breaking change.

**Comment by WhiteAbeLincoln** [comment]
Actually, after changing the css and testing, this doesn't seem to be the issue. I'm not sure what else changed for the sidenav between 0.9.8 and 0.9.7.

**Comment by WhiteAbeLincoln** [comment]
This is fixed in 0.9.8, issue was in 0.9.8-RC1
